appveyor.yml

```diff
@@ -9,32 +9,33 @@ environment:
   matrix:
     # Note: 'default' toolchain is the same as 'vs-14-2015'
 
-    - TOOLCHAIN: "ninja-vs-12-2013-win64"
```

Comment:
This one is lost. I think it's better to comment out `ninja-*` and add new `dummy` to keep things consistent.